collections/_blogs/2020-07-06-my-meshy-journey.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "My Meshy Journey"
+date:   2020-07-03 12:15:05 +0000
+image: /assets/images/posts/2020-07-06-my-meshy-journey/ishita-kumar.png
+author: Ishita Kumar
+#permalink:
+tags: community, open source
+category: contributor
+---
+
+<style type="text/css">
```

Comment:
@chandrashritii, if you can, will you see about pulling these styles into the _sass/blogs.scss?